containers/datalab/run.sh

```diff
@@ -65,6 +82,7 @@ fi
 docker run -it --entrypoint=$ENTRYPOINT \
   -p 127.0.0.1:8081:8080 \
   -v "$CONTENT/datalab:/content/datalab" \
+  $PYDATALAB_MOUNT_OPT \
```

Comment:
That other line is not quite the same: it takes a snapshot of pydatalab and uses it when building the container. Once that snapshot has been taken, changes to the original pydatalab dir have no affect on the container. This new --pydatalab option essentially overwrites whatever pydatalab is in the container with the specified pydatalab when the container starts, but if you then rerun the container without the option, you will be back to whatever pydatalab the container was built with.

Having said that, I don't do enough pydatalab development to know if that old mode might still be useful, so I am OK either way. Given this explanation, do you think I should remove the old approach?